### PR TITLE
Adding initial content for HA and Docker setup

### PR DESCRIPTION
**Pull request by matzew**
This PR shows how to run UPS on different server configurations, like

* Adding some content about running UPS in a WF cluster:
 * WF standalone(full)ha mode
 * WF Managed Domain mode

* Docker:
 * our own Docker image
 * just using Docker for DBs (handy during development)

### CODE REVIEW

**Comment by matzew** [comment]
@danielpassos @abstractj @sebastienblanc mind to review?

@ctomc mind to check it too ?

File: docs/unifiedpush/ups_userguide/server-installation_advanced.asciidoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+For the standalone configuration option of WildFly it is also possible to enable clustering, using the different _ha_ configuration options:
+
+* standalone-ha.xml
+* standalone-full-ha.xml
+
+Since the UnifiedPush Server requires a full profile, we need to start the server using the the _standalone-full-ha.xml_ file:
+
+[source,c]
+----
+$ ./path/to/NODE_x/bin/standalone.sh --server-config=standalone-full-ha.xml -Djboss.node.name=NameOfTheNode
```
Comment (ctomc) [inline docs/unifiedpush/ups_userguide/server-installation_advanced.asciidoc]: you could just use -c standalone-full-ha.xml

**Comment by ctomc** [comment]
looks good :+1:

**Comment by danielpassos** [comment]
:+1: lgtm

**Pull request merged**